cli/types/cypress.d.ts

```diff
@@ -130,6 +130,53 @@ declare namespace Cypress {
     unsupportedVersion?: boolean
   }
 
+  interface Ensure {
```

Comment:
super-nit: could we ship these types in the same order we export?

Missing types for 
- isStrictlyVisible
- exists
- elExists
- isScrollable
- commandCanCommunicateWithAUT

Unless any were meant for internal use